testcharms/charm-repo/bundle/lxd-profile-fail/bundle.yaml

```diff
@@ -0,0 +1,4 @@
+services:
+    lxd-profile-fail:
+        charm: cs:~juju-qa/bionic/lxd-profile-fail-0
+        num_units: 1
```

Comment:
Yes!

Reply:
I realized the suggestion won't do what we want, it should be in a different bundle so that each can fail.  I believe we currently fail on the first validation error, so the 2nd won't be tried.